projects/pre-sentence-reports-to-delius/deploy/values-dev.yml

```diff
@@ -4,6 +4,9 @@ env:
   SENTRY_ENVIRONMENT: dev
   SPRING_JMS_TEMPLATE_DEFAULT-DESTINATION: Digital-Prison-Services-dev-pre_sentence_report_hmpps_queue
   SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_HMPPS-AUTH_TOKEN-URI: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/oauth/token
+  SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json"
+  SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUER_URI: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/issuer"
```

Comment:
What does this do out of interest?  Usually I've only ever configured the JWKS URI.